dotCMS/src/main/java/com/dotcms/api/system/event/ContentletSystemEventUtil.java

```diff
@@ -24,9 +24,15 @@
  */
 public class ContentletSystemEventUtil {
 
+    private static final String DELETE_EVENT_PREFIX = "DELETE";
     private static final String SAVE_EVENT_PREFIX = "SAVE";
     private static final String UPDATE_EVENT_PREFIX = "UPDATE";
-
+    private static final String ARCHIVED_EVENT_PREFIX = "ARCHIVE";
+    private static final String PUBLISH_EVENT_PREFIX = "PUBLISH";
```

Comment:
I think we should add a prefix in order to avoid future event collisions, something like **CONTENTLE**_XYZ